lib/hls/hls_parser.js

```diff
@@ -2826,55 +2830,62 @@ shaka.hls.HlsParser = class {
       }
     }
 
-    // Default AES-128
-    const keyInfo = {
-      bitsKey: 128,
-      blockCipherMode: 'CBC',
-      iv,
+    const aesKeyInfoKey = [
+      drmTag.toString(),
       firstMediaSequenceNumber,
-    };
+    ].join('-');
```

Comment:
Would it be cleaner to express this as a formatted string?
```ts
const aesKeyInfoKey = `${drmTag.toString()}-${firstMediaSequenceNumber}`;
```

Reply:
Done!